NimbleExtension/Sources/JSONMapper/Extensions/Optional+Extras.swift

```diff
@@ -0,0 +1,39 @@
+//  Optional+Or.swift
+//
+//  Created by Edgars Simanovskis on 27/08/2019.
+//  Copyright © 2019 The 1. All rights reserved.
+//
+
+public extension Optional {
+
+    func or(_ otherOptional: @autoclosure () throws -> Wrapped?) rethrows -> Wrapped? {
+        switch self {
+        case .some(let value):
+            return value
+        case .none:
+            return try otherOptional()
+        }
+    }
+    
+    func or(_ otherWrapped: @autoclosure () throws -> Wrapped) rethrows -> Wrapped {
+        switch self {
+        case .some(let value):
+            return value
+        case .none:
+            return try otherWrapped()
+        }
+    }
+
+    func resolve(with error: @autoclosure () -> Error) throws -> Wrapped {
+        switch self {
+        case .none:                 throw error()
```

Comment:
You removed `vertical_whitespace_between_cases` rule, so enter to the new line is better

Reply:
sorry missed it and fixed it, but I hope you agree that having empty line between each case is ugly which is why I removed the rule